driver-scala/src/test/scala/org/mongodb/scala/ApiAliasAndCompanionSpec.scala

```diff
@@ -45,6 +45,7 @@ class ApiAliasAndCompanionSpec extends BaseSpec {
       "BSONTimestampCodec",
       "CausalConsistencyExamples",
       "ChangeStreamSamples",
+      "CreateIndexCommitQuorum",
```

Comment:
I think a type alias and a wrapped object should be added to the scala library:

So in: package.scala add the type alias:
```
  /**
   * Represents the  commit quorum specifies how many data-bearing members of a replica set, including the primary, must
   * complete the index builds successfully before the primary marks the indexes as ready.
   */
  type CreateIndexCommitQuorum = com.mongodb.CreateIndexCommitQuorum
```

And then add a `CreateIndexCommitQuorum` helper object that wraps the scala methods plus tests. See `ReadPreference.scala` and `ReadPreferenceSpec.scala`

Reply:
I have added the helper object.